docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
     labels:
       com.centurylinklabs.watchtower.enable: '${WATCHTOWER_ENABLED:-false}'
     environment:
+      - RLN_CONTRACT_REPO_COMMIT=${RLN_CONTRACT_REPO_COMMIT:-master}
```

Comment:
we should use `main` right? the rlnv2 repo master branch is called main. unsure why github did this, but some repos master is main. :(

Reply:
I changed the default to be a specific commit in https://github.com/waku-org/waku-simulator/pull/95/commits/c118e37dc2eb72f0b164a480810456ab5878666b